cfme/automate/service_dialogs.py

```diff
@@ -129,6 +129,8 @@ def create(self, *element_data):
                 version.LOWEST: "Add a New Element to this Box",
                 "5.3": "Add a new Element to this Box"})
             sel.wait_for_element(element_form.ele_label)
+            fill(element_form, {'choose_type': "Text Area Box"})
+            fill(element_form, {'choose_type': "Check Box"})
```

Comment:
while creating service dialog , adding element, sometimes the UI doesn't load the fields dependent on drop down "text box" and hence the test fails . This happens only once in a while and only in automation. Does not happen manually.
So to refresh the page Iam selecting different value before selecting "Text Box"